staging/cse/windows/containerdtemplate.toml

```diff
@@ -46,6 +46,8 @@ state = "C:\\ProgramData\\containerd\\state"
       [plugins.cri.containerd.runtimes]
         [plugins.cri.containerd.runtimes.runhcs-wcow-process]
           runtime_type = "io.containerd.runhcs.v1"
+          container_annotations = ["io.microsoft.container.processdumplocation"],["io.microsoft.wcow.processdumptype"],["io.microsoft.wcow.processdumpcount"]
```

Comment:
Could you change to the following please?
container_annotations = ["io.microsoft.container.processdumplocation", "io.microsoft.wcow.processdumptype", "io.microsoft.wcow.processdumpcount"]
pod_annotations = ["io.microsoft.container.processdumplocation", "io.microsoft.wcow.processdumptype", "io.microsoft.wcow.processdumpcount"]

https://github.com/containerd/containerd/blob/669e0786d8c390bb176068c720308d7ac93659a3/pkg/cri/config/config.go#L52 pod_annotations and container_annotations are slice of strings

Reply:
Fixed